### DISCUSSION TITLE
nextcloud client cannot login, sometimes

### DISCUSSION THREAD

**Discussion by HedgeShot** [nextcloud/all-in-one#1436] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Use Nextcloud AIO and Nextcloud client on macOS

### Expected behavior 
connection successful
### Actual behavior 
Nextcloud client shows: No connection to Nextcloud at https://url. Server replied "500 Internal Server Error" to "PROPFIND https://url/remote.php/dav/files/admin/"

### Host OS 
Client on macOS (latest version of the client)

#### Nextcloud AIO version 
latest, started happening recently

#### Current channel 
main

#### Other valuable info 
I have 2 nextcloud servers that I connect to using my laptop. One is running on docker WITHOUT using AIO, the other is using AIO. The client can always connect to the nextcloud only server and fails at least once a day when trying to connect to the AIO server. 
When going on https://url/remote.php/dav/files/admin/ using a browser, it is working and then the client connects. Sometimes, when I try to go on the problematic URL using a browser, I get a basic auth pop up (especially if I remove /files/admin/). Note that going to the homepage from a browser always works and shows me as connected.

**Comment by szaimen** [comment]
Hi, are you using AIO behind a reverse proxy?

**Comment by HedgeShot** [comment]
hi,
no I don't, I use the plain vanilla implementation from this repo.

**Comment by szaimen** [comment]
hm... then moving to discussion as I cannot reproduce this.

**Comment by szaimen** [comment]
Are you maybe using Cloudflare somewhere in the chain?

**Reply by HedgeShot** [reply]
nop, the server is hosted on DigitalOcean

**Comment by szaimen** [comment]
So you also don't run some kind of internet firewall in between?

**Reply by HedgeShot** [reply]
I do use DigitalOcean firewall with the following rules:

```
SSH TCP 22  All IPv4 All IPv6    
HTTP    TCP 80  All IPv4 All IPv6    
HTTPS   TCP 443 All IPv4 All IPv6    
Custom  TCP 8443    All IPv4 All IPv6
```

**Reply by szaimen** [reply]
Looks good to me.

**Comment by szaimen** [comment]
Which addons do you have enabled, how many users are using Nextcloud and what are the servers specs?

**Reply by HedgeShot** [reply]
- 3 users (other 2 are on windows and connected to only one nextcloud instance, no issues reported, but they dont check much...)
- no use of Talk, no use of other add-ons

**Reply by szaimen** [reply]
Thanks and what are the server specs? (How many RAM and how many CPUs?)

**Reply by HedgeShot** [reply]
sorry, 2vCPU (no significant load), 2gb RAM (load at 35%), 60gb SSD

**Reply by szaimen** [reply]
Hm... Looks good to me. Can you check the server load when login is not possible?

**Reply by szaimen** [reply]
Hm, looks good to me. Can you check the server load when login is not possible?

**Comment by szaimen** [comment]
Also can you try to whitelist your clients ip-address using the bruteforcesettings app and check if that makes it work?

**Comment by HedgeShot** [comment, marked as answer]
i haven't had the issue since I posted here... while it was happening every day for a couple of weeks before. Magic :-)

**Reply by szaimen** [reply]
Then lets hope that it doesnt come back :-D
